docs/developers/built-in-functions.md

```diff
@@ -0,0 +1,84 @@
+---
+id: built-in-functions
+title: Built-In Functions
+---
+
+## **Introduction**
+
+Elrond protocol has some built-in functions. A built-in function is a special function that is treated by system smart contract. 
+Calls to these functions are considered `built-in function calls` and are treated different than other smart contract calls.
+
+This documentation is subject to change, but at the time of writing, the current built-in functions were:
+
+- ClaimDeveloperRewards
+- ChangeOwnerAddress
+- SetUserName
+- SaveKeyValue
+- ESDTTransfer
+- ESDTBurn
+- ESDTLocalBurn
+- ESDTLocalMint
+- ESDTNFTAddQuantity
+- ESDTNFTBurn
+- ESDTNFTCreate
+
+## **ClaimDeveloperRewards**
+
+This function is to be used by Smart Contract owners in order to claim the fees accumulated during smart contract calls.
+Currently, the developer reward is set to 30% of the fee of each smart contract call.
+
+```
+ClaimDeveloperRewardsTransaction {
+    Sender: <the owner of the SC>
+    Receiver: <SC address>
+    Value: 0
+    GasLimit: 6_000_000
+    Data: "ClaimDeveloperRewards"
+}
+```
+
+:::note
+The amount of available developer rewards can be viewed via [Get Address](/sdk-and-tools/rest-api/addresses/#get-address) endpoint when using the smart contract as the `bech32Address`. 
+:::
+
+## **ChangeOwnerAddress**
+
+`ChangeOwnerAddress` is the function to be called by a Smart Contract's owner when a new owner is desired. 
+
+```
+ChangeOwnerAddressTransaction {
+    Sender: <the current owner of the SC>
+    Receiver: <SC address>
+    Value: 0
+    GasLimit: 6_000_000
+    Data: "ChangeOwnerAddress@" +
+          "@" + <new owner address in hexadecimal encoding>
+}
+```
+
+## **SetUserName**
+
+`SetUserName` is used to set an username for a given address. The receiver's address has to be the DNS smart contract
```

Comment:
N00b comment: no idea what's "the DNS smart contract for the address" and how is it supposed to work,